docs/reference/glossary.md

```diff
@@ -76,6 +76,12 @@ You can refer to a collection of Objects by additionally specifying one of their
 
 Examples: `user:emilia`, `organization:42`, `document:a4259512-3506-4030-a2c7-4b3fd1e9d199`, `team:devops#managers`
 
+## Permission
+
+A `permission` is a computed permission for a particular object. See [relation](#relation) for defining how relationships are structured.
```

Comment:
You really shouldn't use the word in its definition.

A `permission` is a computed set of relations for a particular object. While API callers can check for objects in relations directly, it is preferred to always check a permission as the ability to compute the set allows for design iteration with breaking the calling interface.